dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/exceptions/ApiExceptionHandler.java

```diff
@@ -36,12 +38,14 @@
 public class ApiExceptionHandler {
 
     @ExceptionHandler(ServiceException.class)
+    @ResponseStatus(value = HttpStatus.INTERNAL_SERVER_ERROR)
```

Comment:
I don't think we should add this anno for `ServiceException` because it's a business exception that is actively thrown as expected. WDYT?

Reply:
+1